org.lflang/src/org/lflang/generator/c/CGenerator.java

```diff
@@ -1879,24 +1834,6 @@ public static String variableStructType(TriggerInstance<?> portOrAction) {
         return portOrAction.getParent().reactorDeclaration.getName().toLowerCase()+"_"+portOrAction.getName()+"_t";
     }
 
-    /**
```

Comment:
These two methods were unused, maybe because they perform trivial operations. I deleted them so that my IDE wouldn't mark them with warnings.